.changeset/quick-numbers-raise.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/techdocs-common': patch
+'@backstage/plugin-techdocs-backend': patch
```

Comment:
Since the config parameters are changed, this should be `minor`. Please also provide a short migration description on how to change the config file.

Reply:
added a part on documentation 

https://github.com/backstage/backstage/blob/b206dfc95c9d0e4d344e7fc505185d310588eb89/docs/features/techdocs/using-cloud-storage.md#L426

thanks :)